### DISCUSSION TITLE
Could not initialize event manager: Invalid version: '/usr/lib/python3.9/site'

### DISCUSSION THREAD

**Discussion by fscats** [feelfreelinux/octo4a#339] [General]
Updated my Nexus phone to latest octo4a and now getting an error on the start up of the Octoprint server:

🐙 [37m2023-01-05 01:22:43,043 - octoprint.startup - INFO - ****************************************************************************** [0m
🐙 [37m2023-01-05 01:22:43,046 - octoprint.startup - INFO - Starting OctoPrint 1.7.2 [0m
🐙 [37m2023-01-05 01:22:43,048 - octoprint.startup - INFO - ****************************************************************************** [0m
🐙 [01;31m2023-01-05 01:22:43,418 - octoprint.startup - CRITICAL - Could not initialize event manager: Invalid version: '/usr/lib/python3.9/site' [0m
🐙 Could not initialize event manager: Invalid version: '/usr/lib/python3.9/site'
🐙 [01;31m2023-01-05 01:22:43,418 - octoprint.startup - CRITICAL - There was a fatal error starting up OctoPrint. [0m
🐙 There was a fatal error starting up OctoPrint.

Anyone know the cause or how to install a valid version of python3.9?

**Comment by fscats** [comment]
Well, I should have checked the Issues area first.  Didnt find the python error there, but ran across [these instructions](https://github.com/feelfreelinux/octo4a/issues/325#issuecomment-1345426359) to fix a new install, which fixed my issue, back up and running.  Now if only I could find an USB C connector that works for charging my phone and connecting the printer.  I have a splitter cable and a USB hub that will charge my phone but not connect to a printer, and another USB hub that will connect the printer but not charge my phone :(

**Reply by greiss122** [reply]
Just ran into the same issue while upgrading the app on my Nexus 7, and your link helped!! Much appreciated.
Re: charging, it wouldn't be the same adapter since yours has a USB C port, but I'm able to charge and connect with this cable MOST of the time on my tablet even though it says it won't work. Good luck! https://www.amazon.com/gp/product/B06XD73SCH/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1
